kex-core/src/main/kotlin/org/vorpal/research/kex/descriptor/converter.kt

```diff
@@ -35,7 +35,7 @@ private val ignoreSyntheticObjects by lazy {
 }
 
 class Object2DescriptorConverter : DescriptorBuilder() {
-    private val objectToDescriptor = IdentityHashMap<Any, Descriptor>()
+    var objectToDescriptor = IdentityHashMap<Any, Descriptor>()
```

Comment:
This is not the best way to make the cache 'resettable'. At least you can make a separate function for it and keep internal cache map private

Reply:
But actually I would say that this change is not necessary at all. I will explain it in more detail in SymbolicTraceBuilder